KarhooSDK/Api/Response/TripInfo.swift

```diff
@@ -111,15 +111,20 @@ public struct TripInfo: KarhooCodableModel {
         self.costCenterReference = (try? container.decode(String.self, forKey: .costCenterReference)) ?? ""
         self.partnerTravellerID = (try? container.decode(String.self, forKey: .partnerTravellerID)) ?? ""
         self.partnerTripID = (try? container.decode(String.self, forKey: .partnerTripID)) ?? ""
-        self.dateBooked = (try? container.decode(String.self, forKey: .dateBooked)) ?? ""
         self.agent = (try? container.decode(Agent.self, forKey: .agent)) ?? Agent()
         self.cancelledBy = (try? container.decode(CancelledByPayer.self, forKey: .cancelledBy)) ?? CancelledByPayer()
         self.serviceAgreements = (try? container.decode(ServiceAgreements.self, forKey: .serviceAgreements))
         
-        let utcDate = (try? container.decode(String.self, forKey: .dateScheduled))
-
-        if let utcDate = utcDate {
-            self.dateScheduled = KarhooNetworkDateFormatter(formatType: .booking).toDate(from: utcDate)
+        let utcDateBooked = (try? container.decode(String.self, forKey: .dateBooked))
```

Comment:
I think that at least some unit tests would be useful here. It's weird that we didn't have any, in the first place. What if we make a mistake in parsing dates, now or later, while making some other changes? There's nothing stopping the mistake to be released in production.

Reply:
the KarhooNetworkDateFormatter has some tests